bindings/nodejs/examples/how_tos/advanced_transactions/advanced_transaction.ts

```diff
@@ -63,7 +63,7 @@ async function run() {
         console.log(`Transaction sent: ${transaction.transactionId}`);
 
         console.log('Waiting until included in block...');
```

Comment:
:trollface:

Reply:
Not really wrong though `'accepted': the transaction is included.`:trollface: